client/components/layout/main-layout/footer/Footer.tsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
-
+import { Link } from 'react-router-dom';
+//TODO: Add Links
 const Footer = () => {
-  return <div>Footer Placeholder</div>;
+  return (
+    <>
+      <div className="left-content">
+        &copy; 2024 - LinkTa, LLC. All rights reserved.
+      </div>
+
+      <div className="right-content">
```

Comment:
Please consider the pros and cons with ui/li in this case: https://stackoverflow.com/questions/13416866/why-should-i-use-ul-li-in-my-html